packages/lesswrong/components/form-components/FormComponentDateTime.tsx

```diff
@@ -19,6 +20,9 @@ const styles = (theme: ThemeType): JssStyles => ({
     '& .rdtPicker': {
       bottom: 30,
     }
+  },
+  timezone: {
+    marginLeft: 4
```

Comment:
I would prefer if this was `1 em` because that's roughly the amount of space there would be between these `spans` if it was all one element.

Reply:
I think `1em` is too much space:

<img width="264" alt="Screen Shot 2021-11-23 at 6 53 21 PM" src="https://user-images.githubusercontent.com/9057804/143147034-ce89e32d-70ee-4944-9300-cc09c59e87d8.png">

So I left it how I originally had it. Also good idea to use the `date` for the timezone, it's a nice improvement! :)